avocado/utils/linux_modules.py

```diff
@@ -39,17 +39,18 @@
 BUILTIN = 2
 
 
-def load_module(module_name):
+def load_module(module_name, parameters=''):
     """
     Checks if a module has already been loaded.
     :param module_name: Name of module to check
+    :param parameters: Additional parameters to load module with
     :return: True if module is loaded, False otherwise
     :rtype: Bool
     """
     if module_is_loaded(module_name):
         return False
 
-    process.system('/sbin/modprobe ' + module_name)
+    process.system('/sbin/modprobe %s %s' % (module_name, parameters))
```

Comment:
Well this is a bit tricky as it will only work when the module is not loaded and this will silently proceed giving the wrong impression it worked well. I'd recommend splitting it into 2 branches, if no params are specified, simply run the modprobe, otherwise:

1. check if module is already loaded, if not, modprobe with params
2. check if params in `/sys/modules/$module/params/...` are equal to params, if they are, return
3. try to change the params via sys, return on success
4. try to unload/load the module with new params, return on success
5. raise exception as it's not possible

I have some doubts about (4) as it can de-stabilize the system, so perhaps we can just raise exception instead.